spanner/changestreams/README.md

```diff
@@ -10,7 +10,7 @@ This sample requires [Java](https://www.java.com/en/download/) and [Maven](http:
     [Click here](https://console.cloud.google.com/flows/enableapi?apiid=spanner.googleapis.com&showconfirmation=true)
     to visit Cloud Platform Console and enable the Google Cloud Spanner API.
```

Comment:
```suggestion
    to visit Cloud Console and enable the Google Cloud Spanner API.
```

In 2022 Google Cloud Platform was renamed to Google Cloud for simplicity ([source](https://cloud.google.com/blog/topics/developers-practitioners/introducing-new-homepage-google-cloud))